arches/app/etl_modules/bulk_data_deletion.py

```diff
@@ -129,19 +130,34 @@ def get_sample_data(self, nodegroup_id, resourceids):
 
         return sample_data[0:5]
 
-    def delete_resources(self, userid, loadid, graphid, resourceids):
+    def delete_resources(self, userid, loadid, graphid, resourceids, verbose=False):
         result = {"success": False}
-        user = User.objects.get(id=userid)
+        deleted_count = 0
+        try:
+            user = User.objects.get(id=userid)
+        except ValueError:
```

Comment:
If just confirming whether userid is None, it would be more explicit to avoid a try block and instead do a None check. Something like:
```
user = User.objects.get(id=userid) if userid else None
```

Reply:
Would also save a query. But looks like the proxy model expects `{}` instead of None.

Reply:
fix in latest